docs/3.0rc/guides/deferred-tasks.mdx

```diff
@@ -0,0 +1,309 @@
+---
+title: Deferred Tasks
+description: Examples of using Prefect tasks and the task server.
+---
+
+## Why use deferred tasks?
+
+Prefect tasks are a great way to quickly execute discrete units of work. _Deferred_ Prefect tasks run in a background process using a Prefect task server. This lets you use deferred tasks to move work out of the foreground of your application and distribute concurrent execution across across multiple processes or machines.
+
+For example, if you have a web app, you can use tasks to offload processes such as sending emails, processing images, or inserting data into a database.
+
+Prefect provides a Pythonic interface for defining and running tasks, and this document will focus on how to use defer task execution. However, Prefect tasks support other advanced use cases, such as running tasks in parallel on Ray or Dask clusters, caching return values, configuring automatic retries, and building complex workflows with task dependencies.
+
+## Using deferred tasks
+
+Prefect tasks are Python functions that can be run immediately or submitted for background execution, similar to arq tasks. 
+
+You define a task by adding the `@task` decorator to a Python function, after which you can use the `delay` method to run the task in the background.
+
+If you submit the task for background execution, you'll run a task server in a separate process or container to execute the task. This process is similar to how you would run a Celery worker or an arq worker to execute background tasks.
+
+### Defining a task
+
+Add the `@task` decorator to a Python function to define a Prefect task.
+Here's an example:
+
+```python
+from prefect import task
+
+@task
+def my_background_task(name: str):
+    # Task logic here
+    print(f"Hello, {name}!")
+```
+
+### Calling tasks
+
+You can call a task to run it immediately, or you can defer the task by scheduling it for background execution with `Task.delay`.
+
+**NOTE**: It is also possible to submit tasks to a _task runner_ such as Ray or Dask -- and to defer task execution -- within a workflow, which in Prefect is called a _flow_. However, this document will focus on deferring task execution outside of workflows. For example, by calling `my_task.delay()` within a web application.
```

Comment:
This note is very similar to the paragraph beginning on line 12 above. I think we can do without it.